openrouteservice/src/main/java/org/heigit/ors/routing/RoutingProfileManager.java

```diff
@@ -13,8 +13,8 @@
  */
 package org.heigit.ors.routing;
 
+import com.google.common.base.Strings;
```

Comment:
Wouldn't it be better to use internal Java checks for null and empty string rather than including a google library?

Reply:
replaced all usages of `com.google.common.base.Strings` with a new method in `org.heigit.ors.util.StringUtility`.